.github/CODEOWNERS

```diff
@@ -57,9 +58,12 @@
 /editor/inspector/                            @godotengine/docks
 /editor/scene/                                @godotengine/core
 /editor/scene/2d/                             @godotengine/2d-editor
+/editor/scene/2d/physics                      @godotengine/2d-editor @godotengine/physics
 /editor/scene/3d/                             @godotengine/3d-editor
+/editor/scene/3d/physics                      @godotengine/3d-editor @godotengine/physics
 /editor/scene/gui/                            @godotengine/usability @godotengine/gui-nodes
 /editor/themes/                               @godotengine/usability @godotengine/gui-nodes
+/editor/translations/                         @godotengine/usability
```

Comment:
Picked usability for the general translation area as AFAIK there isn't a dedicated team for that, but I think it's the closest